CHANGELOG.md

```diff
@@ -5,20 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## 0.7.0 - 2019-02-09
 
 ### Added
 
 * Added `bg` package option with `full`, `print`, and `none` as possible values.
 * Added boolean `layout` package option to control whether the package formats the document on load.
 * Added `nomultitoc` package option to toggle multi-column table of contents.
 * Added `dndbook` document class.
+* Added low-resolution background file as an option.
+* Added Russian localization support.
 * Added keycommands to generate text for melee, ranged, and hybrid (melee or ranged) attacks within monsterboxes. Includes localization support for the various phrases used.
 * Added commands to generate titled sections for map areas and sub-areas, with associated counters and automatic reference labelling (as `area:<title>`).
 * Added commands to help generate spell lists.
 
 ### Changed
 
+* Changed footer scroll to vector drawing.
```

Comment:
"Changed footer scroll…" and "Extracted font definitions…" were part of 0.6.

Reply:
Done.